src/StepFunctionAuthoring/util/index.ts

```diff
@@ -0,0 +1,39 @@
+import { WorkFlowType } from 'src/StepFunctionGraph/StepFunctionUtil';
+import { StephFunctionAuthoringFormState } from 'src/StepFunctionAuthoring';
+
+export const TYPE_OPTIONS = [
+    { label: WorkFlowType.TASK, value: WorkFlowType.TASK },
+    { label: WorkFlowType.CHOICE, value: WorkFlowType.CHOICE },
+    { label: WorkFlowType.SUCCEED, value: WorkFlowType.SUCCEED },
+    { label: WorkFlowType.MAP, value: WorkFlowType.MAP },
+    { label: WorkFlowType.PARALLEL, value: WorkFlowType.PARALLEL }
+];
+
+export const FORM_KEYS = {
+    NAME: 'name',
+    NEXT: 'next',
+    NODE_TYPE: 'nodeType'
+};
+
+export const DEFAULT_FORM_STATE: StephFunctionAuthoringFormState = { name: '', next: '', nodeType: '' };
+
+export const DEFAULT_GRAPH_STATE = { States: {} };
```

Comment:
Should this be states (lower case)?

Reply:
Good call out. No, uppercase is correct -- but since you mentioned it I just added the `StepFunctionJSON` typing here.